msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -694,7 +694,13 @@ The following table provides the response codes:
 
 ## Get app content stage sharing capabilities API
 
-The `getAppContentStageSharingCapabilities` API enables you to fetch the app's capabilities for sharing to meeting stage.
+The `getAppContentStageSharingCapabilities` API enables you to fetch the app's capabilities for sharing to meeting stage. The `getAppContentStageSharingCapabilities` API  enables the app to render the share to stage button. If the app has sharing capabilities, then the button appears.
```

Comment:
The share to stage button will not appear if a meeting participant doesn't have permission to share app content to the meeting stage.

Applications should call this API to enable/disable custom share to stage buttons for a meeting participant in the meeting side panel.